### ISSUE TITLE
Space Marine Devastator Sergeants - incorrect wargear options

### ISSUE THREAD

**Issue by Wulfyn** [BSData/wh40k#3435]
**File/Catalogue:** Warhammer 40,000 v1.1.58 (Android) & Warhammer 40,000 8th Edition v57 (Windows)

**BattleScribe version:** 2.01.21.283 (Android) & 2.01.15.114 (Windows)

**Platform:** Android & Windows

**Dropbox:** No

**Description:**
The Space Marines Devastator Sergeant options are incorrect, and are mistakenly limiting the choices to a single item. In the codex the option reads: "The Space Marine Sergeant may replace his boltgun with items from the Sergeant Equipment list." 

Note that it did not say "an item", it said "items". This means multiple items, but Battlescribe is restricting to a single item.

**Comment by GenWilhelm** [comment]
We can't point to any rules because it's an interpretation of the circumstances. People have looked at the different outcomes that may have been intended and chosen the one that is most likely or makes the most sense.

If what you're supporting is intended, then the devastator sergeant would be able to equip three weapons at the same time, which is out of line with the options available on all other sergeants. It would also extend to him being able to fire three pistols at the same time, which is nonsensical and not something that any other model has the opportunity to do.

So the way people have chosen to interpret these rules is that the bolt pistol counts as one of the two weapons that you are restricted to in the sergeant's wargear list. Why he can't also trade in that pistol is anybody's guess.

**Comment by Wulfyn** [comment]
> We can't point to any rules because it's an interpretation of the circumstances. 

Right, and this is why I have raised it as a bug. Because Battlescribe is advertised as being a product that people can pay for to make it easier to create the armies as written in the rulebooks, and you are admitting that you have changed away from those rules because of your personal preference / interpretation.

But what makes your interpretation any more valid than mine? You may think that this interpretation makes the most sense because it prevents the Devastator Sergeant from having 3 bolt pistols, but I think that my interpretation makes the most sense because it doesn't prevent him from taking a combi-weapon and a melee weapon (all other sergeants can upgrade their shooting weapon and take a melee weapon after all, so your interpretation also makes this Sergeant out of line with all the others).

> So the way people have chosen to interpret these rules

I didn't pay for Battlescribe to have some anonymous "people" change the rules to their personal preference. You've admitted that the current version doesn't match the codex. You've admitted that you have no other rule to base your interpretation on. That makes it a bug. You may not like how the codex is written. It may not make any sense to you. Those are beside the point. I want the product to reflect the actual rules, not someone's preference on what they think it should be. Hence the bug report.

I'm not trying to be aggressive here, just direct for the purpose of brevity and clarity.

**Comment by Wulfyn** [comment]
I have looked at that part of the wargear list entry. You are wrong in your interpretation. If you take the phrase "an item" on the third line of that list to only ever mean a singular item then the sentence "one or more of their weapons for an item" means that a Tactical Sergeant trading in his bolt pistol and boltgun can do so for only 1 item and not 2. He would be trading in his bolt pistol and boltgun for "an item".

That would be the interpretation, in English, that you will be forced to take if you want to read it in that way. Any other interpretation would mean that "an item" would have to be taken in a non-singular generic way (in the same way you might say someone lives in a house, even if they own more than one).

**Comment by amis92** [comment]
Please take a read at https://english.stackexchange.com/a/13286

It's based on Oxford Dictionary. If you still have questions as to how to understand that part of the sentence, I'll gladly analyze any other complaints/supporting arguments.

**Comment by Wulfyn** [comment]
Thanks for the reply. However I am confused about what the intent of your message was, so I think we are talking at cross purposes.

My point is that the structure of the wording "an item" in the linked picture does not necessarily imply a singular item despite it having a singular indefinite article because that can be used in a general context. For example if I asked you "Do you have a car?", and you have 2 cars what do you answer? If you say "Yes I have a car", does that mean that I should feel that you lied to me when I find out you have 2 cars and not one car?

I am not sure what point you are driving at in your link. For a start the link is talking about verbs, but I don't recall a verb ever being an issue in the discussion. I'm talking about the general vs specific use of "an item", which is a noun and not a verb. Could you tell me which verb you are talking about, because I think that if anything this is a 'an item vs items' discussion, and not a 'is vs are' discussion.

Secondly your link is demonstrating that in English grammar a single or plural use can be used when the situation is the opposite. In this case a singular verb used for multiple articles in a list. In their example the multiple items "sandwich or a snack" use the singular use of the verb "is" rather than "are". If anything that would strengthen my point if this were about a verb rather than a noun.

And none of the things you have mentioned I can see over-rules the very specific phrasing that a Devastator Sergeant may replace his boltgun with items from the Sergeant Equipment list. That is 100% clear that more than one item can be taken. Why would it say "items" rather than "an item"?

**Comment by OftKilted** [comment]
@Wulfyn Because the verbiage used is identical to ALL of the other Sgts that replace MULTIPLE items (e.g. Boltgun and Pistol) for items from the Sgt list. 

It is a situation where GW did a copy-paste and didn't follow through on their verbiage update.

Please open an email to 40KFAQ@gwplc.com to request an FAQ clarification on their bad verbiage.

**Comment by amis92** [comment]
@Wulfyn 

> 1. (according to you) it is identical to "ALL" of the other Sergeants;
> 2. (according to GenWilhelm) it is out of line with all other Sergeants.
> 
> Which is it?

It's both. You asked:

>Why would it say "items" rather than "an item"?

@OftKilted responded rather precisely: 

> Because the verbiage used is identical to ALL of the other Sgts that replace MULTIPLE items

So it's out of line with all other Sergeants (because all other that replace single weapon use "an item") and in line with Sgts that replace two weapons (because they use "items").

We will add this to #2 (Assumed Amendments), simply because it just doesn't make any sense the current way, due to the note below.

---

**Note**:
I'm still standing firm that the sentence in Lists box ("exchange one or more weapons for an item") is a clear restriction that one weapon is exchanged for 1 item from list.

> I am not sure what point you are driving at in your link. For a start the link is talking about verbs, but I don't recall a verb ever being an issue in the discussion. I'm talking about the general vs specific use of "an item", which is a noun and not a verb. Could you tell me which verb you are talking about, because I think that if anything this is a 'an item vs items' discussion, and not a 'is vs are' discussion.

My link to StackExchange in previous post supports my argument. "one or more weapons" is an abstract non-empty enumeration which the verb "exchange" connects to the "for an item". Now, as you've correctly pointed out, we cannot interpret it that all exchanged weapons result ever in a single item from Lists. This is due to both semantical meaning of this construct, and consistent usage of "items" across all options that replace *multiple* weapons. So, with that interpretation crossed out, we are left with only a single other for that specific syntax (exchange one or more for a sth) - and that "exchange" binds single element from "one or more weapons" to single element from "for an item", therefore resulting in "n:n"/"1:1" constraint.

**Please note that this decision is final unless an FAQ/Errata dictates a change.** No further discussion on this topic will change the outcome. On our part, we submitted a query for an errata to fix "items" for "an item". You're welcome tu submit a query for answer that would change current meaning, or clarify 1:1/n:n for your own comfort.

PS. Please remember that you're always free to make up your own set of files, which may include any and all changes you'd deem necessary.

**Comment by Wulfyn** [comment]
> My link to StackExchange in previous post supports my argument. "one or more weapons" is an abstract non-empty enumeration which the verb "exchange" connects to the "for an item".

"exchange" does not link to "an item" (which is the object of the sentence), but to "models in a unit" (which is the subject of the sentence).

In the webpage that you linked they use these examples:

* a sandwich or other snack is included in the price
* a sandwich and a cup of coffee are included in the price

In these situations the sandwich (etc.) are the subject of the sentence not the object. It is the subject that defines the correct use of the verb, not the object. Also it requires the correct use of the auxiliary verb 'to be' (i.e. is vs are). For your example to be relevant we'd have to have "item" as the subject, and be discussing the use of "is exchanged" vs "are exchanged".

But I am curious, what would you expect "exchange" to be if it were the other way around as per your link?

> Now, as you've correctly pointed out, we cannot interpret it that all exchanged weapons result ever in a single item from Lists.

You have misunderstood - I am saying that if you take "an item" to only ever mean a singular item (as opposed to an abstract generic form), then that sentence has to mean that you will exchange multiple items for just a single one. That's the result as per your argument, not mine.

My argument is that "an item" used in this context is a generic reference. When talking about all items of a class it is customary in English to use the singular indefinite article. For example:

"An apple is an excellent source of vitamin C".

I am clearly not talking about just one apple, despite using the singular term "an apple". Now this can be confusing because you have to look for context to derive the meaning because both use the same wording. But this is not a problem with the sentence that we are discussing, because you have 2 options:

1. Say that "an item" refers to a singular item = this results in your swapping multiple items for just 1 item.
2. Say that "an item" is a generic reference = this results in no limitation about this being 1 item in any circumstance.

Once again you are using your personal subjective interpretation on what you want to be rather than following what the book says. Not a single person here has been able to point at the rule that over-rides the very clear writing that multiple items can be taken. Your wargear link does not say what you think it says, and your grammar link does not apply to this sentence construction.

At this point the only thing justifying your approach is the authority that you have here as a mod. I can't compete with that, and you've made it clear you are unwilling to listen anymore. When I looked at the Battlescribe website and it said to post an error report here if there was an issue in a datafile I actually thought that some progress might occur. That will teach me right?

**Comment by amis92** [comment]
It seems you've focused on my explanation note, instead of an actual resolution, which is adding an Assumed Amendment to #2: https://github.com/BSData/wh40k/issues/2#issuecomment-415371515

If, *taking that amendment into account* (arbitrary but no longer up for discussion), you still feel that a change in datafiles is necessary, please share your reasons.
